xyz

```diff
@@ -143,7 +145,7 @@ echo        # output \n since [enter] output was suppressed
 run() {
   echo "$1"
   if [[ $dry_run == false ]] ; then
-    eval "$1"
+    VERSION="$next_version" PREVIOUS_VERSION="$version" eval "$1"
```

Comment:
Though this is clearly cleaner, it also makes me concerned about triggering some undocumented behaviour in `git` or `node`. Perhaps you could prefix them with `XYZ_`, but that would make this a breaking change. :\

Reply:
I've reverted this change. The diff is much clearer now. Mind taking another look? :eyes: